doozer/doozerlib/util.py

```diff
@@ -591,15 +563,7 @@ def run_oc(auth_file: Optional[str]):
         out, _ = exectools.cmd_assert(cmd, retries=3)
         return json.loads(out)
 
-    # If neither a username nor a password were provided, assume an auth file is being used or no auth at all
-    if not registry_username and not registry_password:
-        return run_oc(auth_file=registry_config)
-
-    # Store the basic auth info in a temporary temp file to be used with --registry-config
-    with tempfile.NamedTemporaryFile(mode='w', prefix="_doozer_") as registry_config_file:
-        registry_config_file.write(get_registry_basic_auth(pullspec, registry_password, registry_username))
-        registry_config_file.flush()
-        return run_oc(auth_file=registry_config_file.name)
+    return run_oc(auth_file=registry_config)
```

Comment:
No longer needs a nested `run_oc` function, right? We can flatten this out

Reply:
updated